jscomp/others/belt_Int.ml

```diff
@@ -0,0 +1,50 @@
+(* Copyright (C) 2017 Authors of BuckleScript
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * In addition to the permissions granted to you by the LGPL, you may combine
+ * or link a "work that uses the Library" with a publicly distributed version
+ * of this file to produce a combined library or application, then distribute
+ * that combined work under the terms of your choosing, with no requirement
+ * to comply with the obligations normally placed on you by section 4 of the
+ * LGPL version 3 (or the corresponding section of a later version of the LGPL
+ * should you choose to use a later version).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
+
+(** {!Belt.Int}
+    Utililites for Int
+*)
+
+external isNaN : int -> bool = "isNaN" [@@bs.val]
+
+external toFloat: int -> float = "%identity"
+
+external fromFloat: float -> int = "%intoffloat"
+
+external fromString: string -> (_ [@bs.as 10]) -> int = "parseInt" [@@bs.val]
+
+let fromString i =
+  match fromString i with
```

Comment:
Would `Number.isInteger` be a better fit here? Then we can catch non-finite and decimal values

Reply:
I don't think `parseInt` can return anything else than an int or `NaN`. 

```js
parseInt("Infinity") // NaN
parseInt("1.4") // 1
```

Reply:
Oh that's true! I normally use `Number("1.4")` etc. to convert strings, because it doesn't try to be smart when non-number values are passed in. But it does return more special values than `parseInt`.